test/fixtures/es6/transpile.js

```diff
@@ -1,7 +1,7 @@
-"use strict";
-var i = 1;
-var foo = function foo(j) {
-  return i + j + "hi";
+'use strict';
```

Comment:
If you get test failures, first place I'd look is rolling back changes to fixtures (otherwise this is fine)

Reply:
Thanks, I was looking out for the same thing. Looks like so far everything is green.